thoth/storages/graph/postgres.py

```diff
@@ -5968,10 +5968,184 @@ def is_database_corrupted(self) -> bool:
                 _LOGGER.error(e)
                 return True
 
-    def get_bloat_data(self) -> dict:
-        """Get bloat data."""
-        # Reference: https://raw.githubusercontent.com/pgexperts/pgx_scripts/master/bloat/table_bloat_check.sql
+    def get_index_bloat_data(self) -> List[Dict[str, Any]]:
+        """Get index bloat data.
 
+        documenation: https://raw.githubusercontent.com/pgexperts/pgx_scripts/master/bloat/index_bloat_check.sql
+        It returns data for each index/table present in the database.
+
+        Examples:
+        >>> from thoth.storages import GraphDatabase
+        >>> graph = GraphDatabase()
+        >>> graph.get_index_bloat_data()
+        [
+            {
+                'database_name': 'postgres',
+                'schema_name': 'public',
+                'table_name': 'depends_on',
+                'index_name': 'depends_on_entity_id_idx',
+                'bloat_pct': Decimal('11'),
+                'bloat_mb': Decimal('623'),
+                'index_mb': Decimal('5821.516'),
+                'table_mb': Decimal('14053.625'), 
```

Comment:
Are `Decimal` types directly returned by this query?

Reply:
Yes

Reply:
It might be a good idea to cast such values to Python float then.